README.md

```diff
@@ -75,6 +75,10 @@ account.unrelease(:email_marketing, :new_email_flow)
 account.rollout?(:email_marketing, :new_email_flow)
 FeatureFlagger::KeyNotFoundError: ["account", "email_marketing", "new_email_flo"]
 
+# Check feature for a specific account id
+Account.rollout_by_id?([:email_marketing, :new_email_flow], 42)
```

Comment:
I think a better method name and signature would be `Account.released_id?(42, :email_marketing, :new_email_flow)`, that reflects this signature `#released_id(resource_id integer, rollout_key variadic of symbols|strings`